README.md

```diff
@@ -420,8 +420,66 @@ By default, this library doesn't print anything to stdout/stderr, however user c
 ## Register oauth2 providers
 
 Authentication handled by external providers. You should setup oauth2 for all (or some) of them to allow users to authenticate. It is not mandatory to have all of them, but at least one should be correctly configured.
+#### Apple Auth Provider
+For configure this provider user require Apple developer account (without it setting up sign in with Apple impossible). [Sign in with Apple](https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_rest_api) lets users log in to your app using their two-factor authentication Apple ID.
 
-#### Google Auth Provider
+Follow to next steps for configure on Apple side: 
+
+1. At first you need login in to developer account and enable capabilities `Sign in with Apple" for specific App ID (in Certificates, Identifiers & Profiles section ).  Also you need `App ID Prefix` as TeamID value, it's required for configure Apple auth provider.
```

Comment:
Is it possible to add links for instruction steps?

Reply:
links added